src/js/createCase.js

```diff
@@ -29,8 +29,22 @@ function registerProduct() {
   return product?.name;
 }
 
-export function createSupportCase(userInfo, fields) {
+async function getProductHash() {
+  const { store } = spinUpStore();
+
+  const currentApp = store.getState().chrome.appId;
+  const path = `${window.location.origin}${window.insights.chrome.isBeta() ? '/beta/' : '/' }apps/${currentApp}/app.info.json`;
+
+  const appData = await fetch(path)
```

Comment:
I don't realy like mixing `await` with `then`s, using just awaits should be enough.
```JS
const appData = await (await fetch(path)).json();
return `Current app: ${currentApp}, Current app hash: ${appData.src_hash}`);
```